app/src/main/java/ghelani/kshamina/sssc_android_app/entity/Assignment.java

```diff
@@ -0,0 +1,40 @@
+package ghelani.kshamina.sssc_android_app.entity;
+
+import android.arch.persistence.room.ColumnInfo;
+import android.arch.persistence.room.Entity;
+import android.arch.persistence.room.ForeignKey;
+import android.arch.persistence.room.PrimaryKey;
+
+import static android.arch.persistence.room.ForeignKey.CASCADE;
+
+@Entity(foreignKeys = {
```

Comment:
Honest question: is this backwards? Deleting a course should delete its weights and assignments, but deleting a weight/assignment should *not* delete its course. Double check that the foreign key belongs here, and not in Course.java